pkg/authorization/controller/authorizationsync/origin_to_rbac_clusterrole_controller.go

```diff
@@ -75,6 +75,9 @@ func (c *OriginClusterRoleToRBACClusterRoleController) syncClusterRole(name stri
 		return c.rbacClient.ClusterRoles().Delete(name, nil)
 	}
 
+	// normalize rules before conversion so RBAC's case sensitive authorizer will work
+	NormalizePolicyRules(originClusterRole.Rules)
```

Comment:
This is mutating a cache.   Seems like you'd do this on the RBAC rule after you converted and copied.